src/components/BuildConfigCreateEditPage/BuildConfigCreateEditPage.tsx

```diff
@@ -206,6 +234,12 @@ export const BuildConfigCreateEditPage = ({ isEditPage = false }: IBuildConfigCr
   );
 
   const submitCreate = (data: IFieldValues) => {
+    setBuildConfigCreatingLoading(true);
+
+    // reset previous results
+    setBuildConfigCreatingError(undefined);
+    setBuildConfigCreatingFinished(undefined);
```

Comment:
These lines could be present also on SCM create page (in submitCreate function).

Reply:
I created new jira to investigate and refactor custom states, NCL-8445